### PR TITLE
Bump version to 0.4.6 and update changelog

### PR DESCRIPTION
**Pull request by dominikl**
See title.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+0.4.6 (June 2019)
+-----------------
+- fixed bug in Image.getPixelValues() method which returned
+  distorted array of pixel values for images with non-square
+  dimensions
```
Comment (sbesson) [inline CHANGELOG.md]: Is it worth mentioning https://github.com/ome/rOMERO-gateway/pull/68 in the notes as well or are you trying to keep this changelog mostly focused on user-facing improvements?

Comment (dominikl) [reply CHANGELOG.md]: Was thinking about it too. But it doesn't really have anything to do with `romero.gateway` package itself, so not necessary to mention it in the changelog I think.

Comment (jburel) [reply CHANGELOG.md]: It is still a useful feature to mention so people know it is there if they want to try it out without having to install anything locally

**Comment by dominikl** [comment]
Travis is happy:
```
*** Welcome to rOMERO 0.4.6 (~ OMERO 5.4) ***
Testing romero.gateway
...
OK:       160
Failed:   0
Warnings: 0
Skipped:  0
```

**Pull request merged**